docs/source/index.rst

```diff
@@ -28,20 +28,16 @@ If you notice any issues, please let us know `on GitHub <https://github.com/ihme
 Introduction
 ------------
 
-The University of Washington IHME Simulation Science Team is excited to introduce pseudopeople, the Python package that simplifies Entity Resolution (ER) research and development.
-This package generates large-scale, simulated population data according to specifications by the user, to replicate a range of complexities of real applications of probabilistic record linkage software.
+The University of Washington IHME Simulation Science Team is excited to introduce pseudopeople, 
+the Python package that simplifies Entity Resolution (ER) research and development.
+This package generates large-scale, simulated population data according to specifications by the user, 
+to replicate a range of complexities of real applications of probabilistic record linkage software.
```

Comment:
```suggestion
to replicate a range of complexities in real applications of probabilistic record linkage software.
```
Not sure if this is better, but when I read the existing version, it sounded weird because of too many "of"s in a row.